src/runtime/contrib/clml/clml_runtime.cc

```diff
@@ -581,9 +583,6 @@ class CLMLRuntime : public JSONRuntimeBase {
 
   bool ExtensionStringPresent(void) {
     cl_int result = 0;
-    if (workspace->platform_id == nullptr) {
-      return 0;
-    }
     size_t reqd_size = 0;
     cl_device_id device_id = workspace->devices[workspace->GetThreadEntry()->device.device_id];
```

Comment:
For now, you don't check that the OpenCL device is available. So it might be a runtime error. I suppose that you can add a new method to `OpenCLWorkspace` e.g.:
```c++
bool IsDeviceExists(int device_id) {
    return device_id < devices.size();
}
```

In this case, you'll be able to softly check if OpenCL device available. And use this method instead of removed if condition. 

Also, you can use `GetCLDeviceID` for getting `device_id`:
```suggestion
    cl_device_id device_id = GetCLDeviceID(workspace->GetThreadEntry()->device.device_id);
```